### ISSUE TITLE
Error with VTK and bin folder

### ISSUE THREAD

**Issue by Huynh-Anh** [valette/FROG#1]
Hi, 
I have done these commands :
git clone https://github.com/valette/FROG.git
cd FROG
git submodule init
git submodule update
cmake . -DCMAKE_BUILD_TYPE=Release
make

however when I get to the cmake part, theres an error with VTK, and when I resolved it, I get in error with the bin folder, for some reason it is empty. Also is it normal that run.sh searches surf3d in bin folder when it is in vtkOPENSURF3D ?

thank you for your help

**Comment by valette** [comment]
Can you provide more information? What is your OS, VTK and cmake versions?
What error appeared with cmake?

**Comment by Huynh-Anh** [comment]
![image](https://user-images.githubusercontent.com/35234402/96931995-2f82a480-148c-11eb-958c-467f10e7cd0d.png)

I have cmake_minimum_required(VERSION 3.10) at the beginning of CMakeList.txt
Description:	Ubuntu 18.04.5 LTS
Release:	18.04

if i do pip list, I cant find VTK or cmake.. is it normal ?

ty

**Comment by valette** [comment]
for c++ projects, you cannot install vtk with pip, but with the package manager. Try:
```
sudo apt-get install libvtk7-dev libopencv-dev libboost-dev
```
this should install boost, opencv and vtk which are needed for frog

Then try to compile the code again (starting from an empty directory would be better)

**Comment by Huynh-Anh** [comment]
I get unable to find package libvtk7-dev
I did sudo apt-get update but I get the same error

my mistake, I am using 
Ubuntu 16.04.6 LTS
Release:	16.04

on my personal computer. I have check online i can seem to find a solution for that package. is there an alternative ?

**Comment by valette** [comment]
OK, Ubuntu 16.04 is a relatively old distribution...
maybe try to replace libvtk7-dev with libvtk-dev

**Comment by Huynh-Anh** [comment]
Hi,
so I have successfully downloaded the dependencies, but i get multiples errors with the make command.
![image](https://user-images.githubusercontent.com/35234402/96936457-17af1e80-1494-11eb-8a22-db0c063a0ddb.png)

and bin folder is empty...

**Comment by Huynh-Anh** [comment]
HI, 
i just want to say that i have managed to resolved the issue but upgrading to Ubuntu 18.04 and manually installing vtk dependencies.

ty again !
